src/modules/APIs.js

```diff
@@ -2,11 +2,12 @@ import nutrients from './food.js';
 
 const foodApi = 'https://themealdb.com/api/json/v1/1/search.php?s';
 
+const commentLink = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/HJ3nJrQxKFWMKnh6RtwV/comments';
 const likeUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/HJ3nJrQxKFWMKnh6RtwV/likes';
 const mealApiObj = [];
 
 nutrients.forEach((food) => {
   mealApiObj.push(`${foodApi}=${food}`);
 });
 
-export {likeUrl, mealApiObj};
+export { likeUrl, mealApiObj, commentLink };
```

Comment:
Hello @Ambient567 you did an amazing job with the way you added the API comments